### PR TITLE
Add cancellation API

### PR DESCRIPTION
**Pull request by 2e3s**

### CODE REVIEW

File: src/Services/SubscriptionService.php [hunk 1]
```diff
@@ -95,23 +94,6 @@ public function update($subscriptionId, $data)
...
 
-    /**
-     * @param string $subscriptionId
-     * @param array|JsonSerializable|SubscriptionCancel $data
-     *
-     * @throws UnprocessableEntityException The input data does not valid
-     *
-     * @return Subscription
-     */
-    public function cancel($subscriptionId, $data)
```
Comment (dara123) [inline src/Services/SubscriptionService.php]: it will break merchant integration when they update SDK?

Comment (2e3s) [reply src/Services/SubscriptionService.php]: Yes.

**Comment by XaosSintez** [comment]
Please add test for `LineItem`

**Comment by 2e3s** [comment]
@XaosSintez did you mean the one I've added?

**Pull request merged**